src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1370,6 +1371,16 @@ BoundExpression bindDefaultArgument(SyntaxNode syntax, ParameterSymbol parameter
                     var memberName = containingMember.GetMemberCallerName();
                     defaultValue = new BoundLiteral(syntax, ConstantValue.Create(memberName), Compilation.GetSpecialType(SpecialType.System_String)) { WasCompilerGenerated = true };
                 }
+                else if (callerSourceLocation is object && getArgumentIndex(parameter.CallerArgumentExpressionParameterIndex, argsToParamsOpt) is int argumentIndex &&
+                    argumentIndex > -1 && argumentIndex < arguments.Length)
+                {
+                    // - Test extension methods.
```

Comment:
Please add a prototype marker here

Reply:
Added the test and removed the comment. It did work without any extra efforts 😄 🎉